### PR TITLE
formatting and broken link fix

### PR DESCRIPTION
**Pull request by jtwilson0294**

### CODE REVIEW

File: src/pages/linux/some-usages-for-linux/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Some Usages for Linux
+How Linux Is Used
```
Comment (bla-ke) [inline src/pages/linux/some-usages-for-linux/index.md]: Change this line to:
```
title: How Linux Is Used
```

Comment (jtwilson0294) [reply src/pages/linux/some-usages-for-linux/index.md]: Thank you! I think I fixed it. If not please let me know (trying to learn Github :D )

Comment (bla-ke) [reply src/pages/linux/some-usages-for-linux/index.md]: No problem :)
Yeah, you fixed it, Travis CI ate this code without any errors, ahahah! I'm also trying to learn GitHub :blush: 

Greetings, Daria

**Comment by Asjas** [comment]
Hey @jtwilson0294 

I'd love to be able to merge your changes but it looks like there is an error with the Travis CI build. ⚠️

Due to no response to the Review Requested I will be marking this PR as stale.

Once you resolve these issues, I will be able to review your PR and merge it. 😊

---

> Feel free to reference the [Article Style Guide](https://github.com/freeCodeCamp/guides#article-title) for this repo on formatting an article correctly so your Travis CI build passes. ✅
>
> Also, it's good practice on GitHub to write a brief description of your changes when creating a PR. 📝
Unfortunately your PR is failing the Travis Build. Please refer to our [Style Guide](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#article-style-guide) on how a article should look.

**Pull request closed without merging**